### PR TITLE
Improve multiselect

### PR DESCRIPTION
**Pull request by CarlSchwan**
- Add focus effect for better accessibility
- Increase font size to be 15px like the other component
- Update style to be more like the NcInputField

There are still some remaining issues that are not fixed by this PR,
notably the width of the multiselect component change when clicking
on it. This is a preexisting issue :(

image

### CODE REVIEW

**Review by jancborchardt — APPROVED** [review]
Looks good!
It's kinda strange that the field itself is double height when open, but guess also preexisting issue.

**Comment by CarlSchwan** [comment]
> Looks good! It's kinda strange that the field itself is double height when open, but guess also preexisting issue.

Yeah here is a before screenshot

image

File: src/components/NcMultiselect/index.scss [hunk 1]
```diff
@@ -165,18 +152,38 @@
...
-			height: 100% !important;
 			border: none;
 			/* override hide to force show the placeholder */
 			/* only when not active */
 			cursor: pointer;
 			/* override inline styling of the lib */
-			padding: 7px 6px !important;
+			padding: 7px 4px !important;
+			display: none;
+            font-size: var(--default-font-size);
```
Comment (CarlSchwan) [inline src/components/NcMultiselect/index.scss:162]:
```suggestion
		font-size: var(--default-font-size);
```

File: src/components/NcMultiselect/index.scss [hunk 2]
```diff
@@ -165,18 +152,38 @@
...
+			height: 40px !important;
+		}
+
+		/* multiselect__limit hidden if active */
+		.multiselect__limit {
 			display: none;
 		}
 	}
 
+
```
Comment (CarlSchwan) [inline src/components/NcMultiselect/index.scss:186]:
```suggestion
```

**Review by max-nextcloud — APPROVED** [review]
Had a quick pass over the code. Looks legit.

**Pull request merged**